### ISSUE TITLE
{{index}} dynamic keyword not replaced in some loop context

### ISSUE THREAD

**Issue by jess8bit** [dudaster/ele-custom-skin#57]
Using Elementor Pro and ECS Pro 1.1.1.

I designed a Loop Template skin that displays a Custom Post Type, and I'm using the {{index}} placeholder in a Text Editor Widget in this skin to display the CPT position in the loop.

I use this skin as custom layout in the Posts Widget, in two contexts:

1) inside a Page, where the Posts Widget query is set to "my CPT". This page is rendered using the Post Content widget inside a Single Page template. --> this works like a charm.

2) inside an Archive template for a Custom Taxonomy (associated to my CPT), where the Posts Widget query is set to "current query" (so that the loop displays only the CPTs associated with the current custom taxonomy term. --> {{index}} is not replaced.

I hope this was clear ^^

Am I trying to do something that does not makes sense(or is unfeasible by design), or is it something that could be done by improving the plugin?

Please let me know if my use case is not specific/clear enough :)
thanks!

**Comment by dudaster** [comment]
Does it matter if it is cpt?

**Comment by dudaster** [comment]
Also try to see if using Archive Posts Widget solves the problem.

**Comment by jess8bit** [comment]
Hi!

(Upgraded to ECS Pro 1.2.0 meanwhile)

I tried with Archive Posts Widget and it is the same output.

I'm not sure I got your previous comment right: did you mean is it the same output if I loop over classic WP Posts instead of Custom Post Types? Well, my website does not have any classic Posts to test with, but I created a dummy one for that purpose and then in Posts widget I switched the query source from Current Query to Posts but this doesn't change the {{index}} output. 
(And Archive Posts doesn't let you change the query source since it's basically a shortcut to Current Query).
